awx/main/management/commands/run_callback_receiver.py

```diff
@@ -17,21 +19,40 @@ class Command(BaseCommand):
     '''
     help = 'Launch the job callback receiver'
 
+    def add_arguments(self, parser):
+        parser.add_argument('--status', dest='status', action='store_true',
+                            help='print the internal state of any running callback receiver')
+
     def handle(self, *arg, **options):
+        control_routing_key = 'callback_receiver-{}-control'.format(get_local_queuename())
+        if options.get('status'):
+            print(Control(
+                'callback_receiver',
+                host=settings.CALLBACK_QUEUE,
+                routing_key=control_routing_key
+            ).status())
+            return
+
         with Connection(settings.BROKER_URL) as conn:
             consumer = None
             try:
+                queues = [
+                    Queue(
+                        settings.CALLBACK_QUEUE,
+                        Exchange(settings.CALLBACK_QUEUE, type='direct'),
+                        routing_key=settings.CALLBACK_QUEUE
+                    ),
+                    Queue(
+                        settings.CALLBACK_QUEUE,
+                        Exchange(settings.CALLBACK_QUEUE, type='direct'),
+                        routing_key=control_routing_key,
```

Comment:
so the only job of this new `Queue` is to listen for the status command?

_looks up how Queue class functions again_